.mocharc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "exit": true
```

Comment:
Just curious, what does this do?

Reply:
See https://mochajs.org/#-exit.

Reply:
We may want to file an issue for figuring out what in our tests is causing a hang?

Reply:
Filed #2673.